dist/index.js

```diff
@@ -0,0 +1 @@
s[i];descriptor.enumerable=descriptor.enumerable||false;descriptor.configurable=true;if("value"in descriptor)descriptor.writable=true;Object.defineProperty(target,descriptor.key,descriptor);}}return function(Constructor,protoProps,staticProps){if(protoProps)defineProperties(Constructor.prototype,protoProps);if(staticProps)defineProperties(Constructor,staticProps);return Constructor;};}();var _react=require('react');var _react2=_interopRequireDefault(_react);var _reactNative=require('react-native');function _interopRequireDefault(obj){return obj&&obj.__esModule?obj:{default:obj};}function _classCallCheck(instance,Constructor){if(!(instance instanceof Constructor)){throw new TypeError("Cannot call a class as a function");}}function _possibleConstructorReturn(self,call){if(!self){throw new ReferenceError("this hasn't been initialised - super() hasn't been called");}return call&&(typeof call==="object"||typeof call==="function")?call:self;}function _inherits(subClass,superClass){if(typeof superClass!=="function"&&superClass!==null){throw new TypeError("Super expression must either be null or a function, not "+typeof superClass);}subClass.prototype=Object.create(superClass&&superClass.prototype,{constructor:{value:subClass,enumerable:false,writable:true,configurable:true}});if(superClass)Object.setPrototypeOf?Object.setPrototypeOf(subClass,superClass):subClass.__proto__=superClass;}var LinearGradient=function(_PureComponent){_inherits(LinearGradient,_PureComponent);function LinearGradient(){var _ref;var _temp,_this,_ret;_classCallCheck(this,LinearGradient);for(var _len=arguments.length,args=Array(_len),_key=0;_key<_len;_key++){args[_key]=arguments[_key];}return _ret=(_temp=(_this=_possibleConstructorReturn(this,(_ref=LinearGradient.__proto__||Object.getPrototypeOf(LinearGradient)).call.apply(_ref,[this].concat(args))),_this),_this.state={width:1,height:1},_this.measure=function(_ref2){var nativeEvent=_ref2.nativeEvent;return _this.setState({width:nativeEvent.layout.width,height:nativeEvent.layout.height});},_this.getAngle=function(){var angle=Math.atan2(_this.state.width*(_this.props.end.y-_this.props.start.y),_this.state.height*(_this.props.end.x-_this.props.start.x))+Math.PI/2;return angle+'rad';},_this.getColors=function(){return _this.props.colors.map(function(color,index){var location=_this.props.locations[index];var locationStyle='';if(location){locationStyle=' '+location*100+'%';}return color+locationStyle;}).join(',');},_temp),_possibleConstructorReturn(_this,_ret);}_createClass(LinearGradient,[{key:'render',value:function render(){return _react2.default.createElement(_reactNative.View,{style:[this.props.style,{backgroundImage:'linear-gradient('+this.getAngle()+','+this.getColors()+')'}],onLayout:this.measure,__source:{fileName:_jsxFileName,lineNumber:54}},this.props.children);}}]);return LinearGradient;}(_react.PureComponent);LinearGradient.defaultProps={start:{x:0.5,y:0},end:{x:0.5,y:1},locations:[],colors:[]};exports.default=LinearGradient;
```

Comment:
Could you revert this change please - not commit `dist` folder
This is bad practice because it makes git diffs hard to read and beginners might contribute to dist files unknowingly

Reply:
This is the crux of the issue -- the reason I opened this PR. I cannot exclude dist because look at package.json:

```
"main": "dist/index.js"
```

If I exclude it, `main` will point to a file that doesn't exist.